apps/storybook/src/Table.stories.tsx

```diff
@@ -314,9 +314,15 @@ export const Sortable: Story<Partial<TableProps>> = (args) => {
         columns: [
           {
             id: 'name',
-            Header: 'Name',
+            Header: 'Name (sorts ascending first)',
```

Comment:
After changing string, you need to update visual tests

Reply:
Updated